docs: add APIv4 Pulsar storage policies endpoint (#232)
CleverCloud/documentation: content/api/v4.md

- v4
---
Clever Cloud provides a public REST API used by its own services and customers, known as [`/v2`](/api/v2). 
As our platform evolves, we're progressively moving to `/v4` endpoints, adding functionalities, with a more modern approach: 
```
https://api.clever-cloud.com/v4/
```

> While we're still in the process of documenting it, you'll find the more useful endpoints and example requests in this page.

Reviewer: ```suggestion
While the documentation moves towards completion, this page presents the most useful endpoints and example requests available now. 
```

  },
  {...}
]
```

## Pulsar policies
- `/v4/addon-providers/addon-pulsar/addons/{addon-id}/storage-policies`
- Type: `GET`/`PATCH`

> Define or get offload/retention policies of a [Pulsar add-on](/doc/addons/pulsar/). Retention is how long messages are kept in the topic. Offload is how long messages are kept in the hot storage (NVMe SSD) before being moved to Cellar object storage (S3 compatible, HDD).

Reviewer: ```suggestion
Define or get offload/retention policies of a [Pulsar add-on](/doc/addons/pulsar/).
- **Retention** indicates how long the topic retains messages.
- **Offload** indicates how long the hot storage retains messages (NVMe SSD) before they move to Cellar object storage (S3 compatible, HDD).
```
